docs/csharp/misc/cs0193.md

```diff
@@ -10,7 +10,7 @@ ms.assetid: 7b60fd99-9eee-4d61-ad75-585a16e22e96
 # Compiler Error CS0193
 The * or -> operator must be applied to a pointer  
   
- The **\*** or **->** operator was used with a nonpointer type. For more information, see [Pointer types](../../csharp/programming-guide/unsafe-code-pointers/pointer-types.md).  
+ The **\\*** or **->** operator was used with a nonpointer type. For more information, see [Pointer types](../../csharp/programming-guide/unsafe-code-pointers/pointer-types.md).  
```

Comment:
@mairaw will it render correctly? Now it looks like escaping a backslash, which escapes `*`

Reply:
We're validating what the CI script has done. Let me check this one @pkulikov

Reply:
I'll fix this and remove the bold. Don't think it's needed. It's even broken right now on docs. Thanks for bringing this to our attention @pkulikov!